packages/device_info/device_info_platform_interface/lib/model/ios_device_info.dart

```diff
@@ -45,15 +45,15 @@ class IosDeviceInfo {
   /// Deserializes from the map message received from [_kChannel].
   static IosDeviceInfo fromMap(Map<String, dynamic> map) {
     return IosDeviceInfo(
-      name: map['name'] ?? '',
-      systemName: map['systemName'] ?? '',
-      systemVersion: map['systemVersion'] ?? '',
-      model: map['model'] ?? '',
-      localizedModel: map['localizedModel'] ?? '',
-      identifierForVendor: map['identifierForVendor'] ?? '',
+      name: map['name']!,
```

Comment:
One example of tests being difficult :)

https://github.com/flutter/plugins/blob/master/packages/device_info/device_info_platform_interface/test/method_channel_device_info_test.dart

This will pass a map with partially populated data which then will fail the null checks.

Reply:
I added the cases in https://github.com/flutter/plugins/pull/3269/commits/0acdd4b72c4b51142c8424eb306eeabcec90e59b. I think this may also help someone looking for sample outputs. :)